run.go

```diff
@@ -15,32 +15,30 @@
 package terramate
 
 import (
-	"fmt"
-	"os"
 	"os/exec"
 	"path/filepath"
 
-	"github.com/mineiros-io/terramate/project"
 	"github.com/mineiros-io/terramate/stack"
+
+	"github.com/rs/zerolog/log"
 )
 
 func Run(root string, stacks []stack.S, cmdSpec *exec.Cmd) error {
-	wd, err := os.Getwd()
-	if err != nil {
-		return fmt.Errorf("error getting current directory: %w", err)
-	}
-
 	for _, stack := range stacks {
 		cmd := *cmdSpec
 
-		stackdir, _ := project.FriendlyFmtDir(root, wd, stack.Dir)
-		fmt.Fprintf(cmd.Stdout, "[%s] running %s\n", stackdir, &cmd)
+		log.Info().
+			Str("stack", stack.Dir).
+			Str("cmd", cmd.String()).
```

Comment:
@katcipis @i4ki how can we show the user command here instead of the calculated executable?

Reply:
After exec.Cmd is created we don't have that info anymore AFAIK, it already stores on the calculated executable. we would need to log this before when the exec.Cmd is being created (probably being created with: https://pkg.go.dev/os/exec#Command).

Reply:
why is this object created outside of the run function?

Reply:
just to avoid passing stdin, stdout, stderr as arguments to `terramate.Run()`, as they are needed for tests. But you can change this, not big deal. You can change the Run to something like:

```
func Run(root string, stacks []stack.S, command []string, stdin io.Reader, stdout, stderr io.Writer) error
```

or create a run type containing the cmd spec (cmd, args, standard I/O, etc):

```
func Run(root string, stacks []stack.S, cmdSpec YourOwnCmdSpec) error {
```

Reply:
gotcha... for UX it would still be great to show the users command and not the evaluated executable... i will create a ticket so this can be fixed in the future... fine for now, i guess.. ;)